web_client/app/awaiting-confirmation/awaiting-confirmation.tsx

```diff
@@ -36,7 +36,7 @@ const style = styles({
     padding: vars.spacing
   },
   maxWidth: {
-    textAlign: 'center'
+    textAlign: 'center',
```

Comment:
With https://github.com/citizenlabsgr/voter-engagement/pull/29, this will now cause a build failure.

Reply:
You're reviewing stale code; fixed in bff0843.